### PR TITLE
Fixes/deposit tab

### PR DESCRIPTION
**Pull request by IgboPharaoh**
Modified all screens of the deposit tab to be responsive.

### CODE REVIEW

File: src/components/Button.tsx [hunk 1]
```diff
@@ -7,7 +7,7 @@ export type ButtonProps = {
 	size?: string;
 	fontSize?: ResponsiveObject<string>;
 	height?: string | number;
```
Comment (okjodom) [inline src/components/Button.tsx:9]: nit: do we need the same responsive prop type for the height?

File: src/components/DepositTab.tsx [hunk 2]
```diff
@@ -199,20 +209,33 @@ interface ShowDepositAddressProps {
...
+			<Heading
+				fontWeight='500'
+				fontSize={{ base: '22', md: '24' }}
+				color='#1A202C'
+			>
+				Bitcoin Deposit to Federation
+			</Heading>
 
 			<Text py='3'>
 				Please pay to the address address shown <br /> to deposit funds into
```
Comment (okjodom) [inline src/components/DepositTab.tsx:230]: nit: we should look to remove the <br> tags formatting this lead text

File: src/components/Button.tsx (hunk 1 above)
Comment (IgboPharaoh) [reply src/components/Button.tsx:9]: We might in the future, but I don't see any use case at the moment

File: src/components/DepositTab.tsx (hunk 2 above)
Comment (IgboPharaoh) [reply src/components/DepositTab.tsx:230]: ok will look into that

File: src/components/DepositTab.tsx [hunk 3]
```diff
@@ -28,6 +28,13 @@ enum DepositStatus {
 	Complete,
 }
 
+const sliceString = (arg: string): string => {
```
Comment (okjodom) [inline src/components/DepositTab.tsx:31]: nit: `truncateAddress` might be a better name for this utility. Name should show that it doesn't affect the underlying data, just it's visual formatting.

- `formatTrucateAddress`?
- `truncateStringFormat`? - if utility is used for more that truncating addresses alone

**Review by okjodom — APPROVED** [review]

File: src/components/DepositTab.tsx (hunk 2 above)
Comment (okjodom) [reply src/components/DepositTab.tsx:230]: marking as follow up 👍🏿

File: src/components/DepositTab.tsx (hunk 3 above)
Comment (IgboPharaoh) [reply src/components/DepositTab.tsx:31]: Alright

**Pull request merged**